### ISSUE TITLE
trapFocus sticks on close button.

### ISSUE THREAD

**Issue by psherl** [sachinchoolur/lightGallery#1504]
## Description

trapFocus stuck on close button

### Steps to reproduce

Add a lightgallery with the video plugin, and include a single video from youtube, and trapFocus enabled in the settings. Once you tab via keyboard navigation to the close button, either before of after the iframe with the video loads, the focus is trapped onto the close button and you cannot navigate via keyboard to any other links.

I believe this is due to the trapFocus event running after the video has loaded, as well as the associated controls, meaning that not all clickable elements are included within the items within the focus trap .

### JS code that you use to initialize lightGallery.

```js
lightGallery(document.getElementById('js-lightbox'), {
  plugins: [lgVideo],
  trapFocus: true,
  closable: true,
});
```

### Sample HTML markup

```html
<div class="js-lightbox">
    <a class="c-col-embedded-media__cover-link" href="#" data-src="https://www.youtube.com/embed/1ekZEVeXwek">
</div>
```

### Environment

-  lightGallery version - 2.7.1
- multiple browsers
